### PR TITLE
Aditionals

### PR DESCRIPTION
**Pull request by ad9311**
# This is a PR for a review of the Project 3: Using Bootstrap - Milestone 3

In this branch we have the previous approved Milestone 2 ☑️ plus the bottom sections "Subscribe" & "Newsletter"

This project is done by Ángel Díaz @ad9311 and Julián Carracedo @JuliCarracedo

### CODE REVIEW

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
# Status:  ♻️ Changes Required ♻️
 Hi @ad9311  and @JuliCarracedo 

**You've really hit all the requirements in one shot. Well done.** Here're the major highlights of this PR:
- Your PR title and description are really descriptive.
- Used many bootstrap classes.

**However, in order to approve, we have to work a little bit more. Please check my comments below and fix them as appropriate.**

## For Approval

## General Requirements

- [x] All images should have links.
- [x] Use the DRY (Do not repeat yourself) principle wherever possible. I have mentioned some of them in the comments.
 - `.background-hun`, `.background-rank`, and `.background-biden` classes have same `background-size and height` property. You can put them in one class and use them.

## Maganize and Editor's pick

- [x] Both `magazine` and `Editor's pick section` are not looking the same in laptop view. 
- [x] The editor's pick section is not looking the same in any resolution. See the image below.

Screenshot (1316)_LI

- [ ] All section contains duplicate articles, please fix this to match the original. If we use the same content everywhere it will not look like a clone.

Screenshot (1320)

## Subscribe section

- [ ] The text should overlap the book. In your clone, it's not overlapping in mobile view.

Screenshot (1317)_LI

- [x] In laptop view the subscribe section is not looking the same.
  
**Your clone**

Screenshot (1319)

**Original**

Screenshot (1318)

## [Optional]

- [ ] I will recommend using the same red color in all the buttons and everywhere to look like the original website.

Remember, This is optional and nothing to do with the project approval.

## Please make the required changes and submit another review request when you are done.

## Reminder

**"As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) now, you have only 2 code review left for this project."**


(Feel free to contact me on slack by the handle of  **deep** if you have any additional questions).
Happy Coding
[Dipesh Kumar](https://deeptwist.tk) | [Twitter](https://twitter.com/97deepeshkumar)

**Comment by ad9311** [comment]
## The following changes address the previously mentioned issues. ✅
### Necessary changes
- All images have links now
- Removed repeated classes to follow the DRY principle.
- Change all sections to be the same as the original in terms of responsiveness 
- The text in subscribe section now overlaps in the image
- The subscribe section now responds the same as the original in different devices

### Optional changes
- Change the color red tonality to match the original one
- All input tags are now of type "email"

File: index.html [hunk 1]
```diff
@@ -14,9 +14,9 @@
       <nav class="navbar navbar-light container-fluid m-0 p-0">
         <div class="red-background row container-fluid p-0 m-0">
             <p class="d-none d-xl-inline col-3 link-light">Wed, Jan 20, 2021</p>
```
Comment (christianotieno) [inline index.html:16]: ##  Fix the layout of the `footer` and the `navbar`
- Before we proceed, we need to make sure nothing is messed up. Kindly have a look and work on the affected areas.
- [ ] The `footer` and the `Navbar` needs to have the same view width as the body on all screens. let's fix this. 

|Source Page|Clone Page|
|:--:|:--:|
|source|clone|

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @ad9311 @ @JuliCarracedo 👋🏾 
The project is coming along just fine. We still have some changes that might have been produced some bugs with the code. Let's look at them fix them before proceeding, thanks.

### mandatory
- [ ] Fix the layout of the `footer` and the `navbar`

### happy coding 💻 
#### Christian Otieno (CR)

**Comment by ad9311** [comment]
## The following changes address the previously mentioned issues.
- Fix the width of the navbar and footer.

**Review by elisha2kyakpo1 — APPROVED** [review]
**Congratulations guys!*🎉💪✔. Let's drink water, let's celebrate! I'm to see you implement all the changes requested*
I'm wishing you happiness as you move to the next step.
Go ahead and merge.💪🎉👍
**[OPTIONAL] you should remove the spaces so that the body has the same look from header to footer**
Elisha kyakopo.

**Pull request merged**